draft-ietf-quic-manageability.md

```diff
@@ -820,22 +820,40 @@ possible via the observation of CE markings on ECN-enabled QUIC traffic.
 ## Stateful Treatment of QUIC Traffic {#sec-stateful}
 
 Stateful treatment of QUIC traffic (e.g., at a firewall or NAT middlebox) is
-possible through QUIC traffic and version
-identification ({{sec-identifying}}) and observation of the handshake for
-connection confirmation ({{sec-confirm}}). The lack of any visible end-of-flow
-signal ({{sec-teardown}}) means that this state must be purged either through
-timers or through least-recently-used eviction, depending on application
-requirements.
+possible through QUIC traffic and version identification ({{sec-identifying}})
+and observation of the handshake for connection confirmation ({{sec-confirm}}).
+The lack of any visible end-of-flow signal ({{sec-teardown}}) means that this
+state must be purged either through timers or through least-recently-used
+eviction, depending on application requirements.
 
 {{?RFC4787}} requires a timeout that is not less than 2 minutes for most UDP
-traffic.
-However, in pratice, timers are often lower, in the range of 15 to 30 seconds.
-In contrast, {{?RFC5382}} recommends a timeout of more than 2 hours for TCP,
-given that TCP is a connection-oriented protocol with well-defined closure
-semantics. For network devices that are QUIC-aware, it is recommended to also
-use longer timeouts for QUIC traffic, as QUIC is connection-oriented. As such,
-a handshake packet from the server indicates the willingness of the server to
-communicate with the client.
+traffic.  However, in practice, timers are sometimes lower, in the range of 30
+to 60 seconds. In contrast, {{?RFC5382}} recommends a timeout of more than 2
+hours for TCP, given that TCP is a connection-oriented protocol with well-
+defined closure semantics.
+
+Even though QUIC has explicitly been designed tolerate NAT rebindings, decreasing the
+NAT timeout is not recommended, as it may negatively impact application performance
+or incentivize endpoints to send very frequent keep-alive packets.  Instead it is
+recommended, even when lower timers are used for other UDP traffic, to use a
+timer of at least two minutes for QUIC traffic.
```

Comment:
```suggestion
Even though QUIC has explicitly been designed tolerate NAT rebindings,
decreasing the NAT timeout is not recommended, as it may negatively impact
application performance or incentivize endpoints to send very frequent
keep-alive packets. Instead it is recommended, even when lower timers are
used for other UDP traffic, to use a timer of at least two minutes for QUIC
traffic.
```